src/client/auth.ts

```diff
@@ -197,12 +282,14 @@ export async function discoverOAuthMetadata(
  * Begins the authorization flow with the given server, by generating a PKCE challenge and constructing the authorization URL.
  */
 export async function startAuthorization(
-  serverUrl: string | URL,
+  authorizationServerUrl: string | URL,
   {
+    resource,
```

Comment:
```suggestion
```
I think this was an earlier version of the draft? resource isn't included in the current draft.